### PR TITLE
Added architecture descriptions and diagrams

### PR DESCRIPTION
**Pull request by breakpointer**
New architecture drawings and diagrams added. Also made updates to content and structure based on previous feedback.

**Related Issue**  
Supports #78 
Supports #4 

**Related PRs**  
This PR is not dependent on any other PR

**What does this PR do?**  
This PR adds description and some architecture diagrams (draft diagrams) to the Readme. 

**Description of Changes**  
New files under `docs/images` and more content in the readme

**What gif most accurately describes how I feel towards this PR?**  
Me rite guud

### CODE REVIEW

**Comment by mgw-sbic** [comment]
Hi Bootleg-ers! Our company Smart Blonde Experiential would like to be able to incorporate Bootleg functionality into our existing platform, called TotalVU. As a small part of our platform, we use an app that allows people at the concert to create a "shoutcast" of the concert and publish it into the overall set of video streams that make up the live and VoD experience. Would your team be willing to consider our use case in the design of the architecture?

**Comment by humbitious** [comment]
> Hi Bootleg-ers! Our company Smart Blonde Experiential would like to be able to incorporate Bootleg functionality into our existing platform, called TotalVU. As a small part of our platform, we use an app that allows people at the concert to create a "shoutcast" of the concert and publish it into the overall set of video streams that make up the live and VoD experience. Would your team be willing to consider our use case in the design of the architecture?

Hey @mgw-sbic, we'd love to talk about that. Follow and tweet me @jwolpert or us on @web3studio on twitter, and we can follow back and arrange for a conversation.

**Comment by breakpointer** [comment]
@barlock How's it looking now? I've made some more changes. Can I get a 👍 ?

File: README.md [hunk 1]
```diff
@@ -23,7 +24,10 @@ latest updates here through GitHub. Please join the discussion, create issues, a
 
 Read more about the project in the latest article about Bootleg on [Medium](https://link.medium.com/ZgnzmtPMrT)
 
-<br/>
+
+## Project overview
+
+This repo holds the code and development assets for the Bootleg SDK. The repo and many of the high level concepts in this project all pay homage to the original article about the idea published on Medium (link above). Whenever possible we have attempted to use more generic terms in our code and domain models so that as much code from this repo can be re-used, shared, re-purposed by anyone. We have done our best but it is a tradeoff between making something we feel embodies the spirit of the idea and speculating about would/would not be useful should someone decided to take our code and run with it.
```
[the remark this answers is not in the document]
Comment (breakpointer) [reply README.md]: Ah, okay. That's cool. I was not aware you could do named links in MD.

**Comment by termux-io** [comment]
`https://patch-diff.githubusercontent.com/raw/ConsenSys/web3studio-bootleg/pull/6.diff`

**Pull request merged**